src/main/java/com/codeborne/selenide/CollectionCondition.java

```diff
@@ -11,9 +11,9 @@
 import com.codeborne.selenide.collections.TextsInAnyOrder;
 import com.codeborne.selenide.impl.WebElementsCollection;
 import com.google.common.base.Predicate;
-import org.checkerframework.checker.nullness.compatqual.NullableDecl;
 import org.openqa.selenium.WebElement;
 
+import javax.annotation.Nullable;
```

Comment:
@asolntsev Maybe it's will be better to import `org.checkerframework.checker.nullness.qual.Nullable;` to match implemented method signature from guava library?